README.md

```diff
@@ -1,59 +1,56 @@
-Heureka ověřeno PHP API
-=======================
+Heureka 'Ověřeno zákazníky' PHP API
+===================================
 
-[Heureka Ověřeno](http://overeno.heureka.cz/) service API for PHP. 
+[Heureka Ověřeno zákazníky](http://overeno.heureka.cz/) (ShopCertification) service API client implementation for PHP.
+
+Examples
+--------
+
+You can check working examples in the folder `examples` of this repository.
 
 Usage
 -----
 
-Initialize Service using [your API key](http://sluzby.heureka.cz/sluzby/certifikat-spokojenosti/):
+Initialize class `Heureka\ShopCertification` using
+[your API key](http://sluzby.heureka.cz/sluzby/certifikat-spokojenosti/):
 
 ```php
 require_once __DIR__ . '/vendor/autoload.php';
-$overeno = new \Heureka\Overeno('xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx');
-```
-      
-SK shops should initialize Heureka Overeno service with second parameter \Heureka\Overeno::LANGUAGE_SK:
-      
-    $overeno = new \Heureka\Overeno('xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx', \Heureka\Overeno::LANGUAGE_SK);
-      
-Set customer email:
 
-```php
-$overeno->setEmail('jan.novak@example.com');
+$shopCertification = new \Heureka\ShopCertification('xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx');
 ```
-  
-Add product from order - encoded in UTF8 if possible. Service can handle WINDOWS-1250 and ISO-8859-2 if necessary  
+
+**Keep in mind that your API key is only yours and it is supposed to be a secret.** Never post your API key to anyone,
+never put it into JavaScript or anywhere else. It should live on your server only. If you feel the need to break this
+rule then you are doing something wrong - please consult our supported department prior to any actions.
+
+SK shops should initialize the class with a service parameter in the options:
 
 ```php
-$overeno->addProduct('Nokia N95');
+$options = ['service' => \Heureka\ShopCertification::HEUREKA_SK];
+$shopCertification = new \Heureka\ShopCertification('xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx', $options);
 ```
 
-Or add multiple products:
+Set the customer e-mail address:
 
 ```php
-// array $products is populated elsewhere by shop application
-foreach ($products as $product) {
-  $overeno->addProduct($product);
-}
+$shopCertification->setEmail('jan.novak@muj-eshop.cz');
 ```
-    
-or/and add products using [item ID](http://sluzby.heureka.cz/napoveda/xml-feed/#ITEM_ID):
+
+Set the customer's order ID (only integers are allowed):
 
 ```php
-$overeno->addProductItemId('B1234');
+$shopCertification->setOrderId(15195618851564);
 ```
-  
-Provide order ID - BIGINT (0 - 18446744073709551615):
 
+Add products which the customer ordered:
```

Comment:
Co spis "Add products which the customer ordered using ITEM_ID (your products ID):"